include/deal.II/grid/grid_tools.h

```diff
@@ -157,10 +158,27 @@ namespace GridTools
    * get the same result using <code>cell-@>measure()</code>, but this
    * function also works for cells that do not exist except that you make it
    * up by naming its vertices from the list.
+   *
+   * Instantiations of this function are available for the <tt>dim !=
+   * spacedim</tt> cases, but these just raise exceptions. They are only
+   * provided as an aid for generic programming.
+   */
+  template <int dim, int spacedim>
```

Comment:
I am sure that the number of people would could use C+11 is actually very close to 100%. People that don't have C++11 are usually on a cluster with centos 6 and they don't know how to load a module with a newer compiler. So they use the default compiler which is gcc 4.4